explorer/fuzzing/BUILD

```diff
@@ -110,6 +110,7 @@ cc_fuzz_test(
     name = "explorer_fuzzer",
     size = "small",
     srcs = ["explorer_fuzzer.cpp"],
+    copts = ["-Wno-missing-prototypes"],
```

Comment:
Why this approach here, versus what's done for other fuzzers?

Reply:
Thanks for explaining, would it be worth adding a small comment to this effect?

Reply:
Adding comment.